client/src/components/UserAvatar.js

```diff
@@ -58,7 +58,7 @@ const UserAvatar = () => {
 					</Menu.Item>
 					<Menu.Item>
 						{({ active }) => (
-							<Link onClick={() =>
+							<Link to="#" onClick={() =>
```

Comment:
If it's redirecting to the same page, then the component `Link` shouldn't be used at all. This seems more like a `button` to me